### PR TITLE
Fix: documentation update for version 3.0

### PR DESCRIPTION
**Pull request by Savokr**
Added documentation for reveal 3.x as separate version in docusaurus.
Several questions that should be solved:
1. Upload newly processed models for version 3.x or change ids if they are already available.
2. Sometimes you need to reload the page because of authentication issue.
3. Do we want to remove the banner that says that 2.x is not actively maintained? (turned out to be not that easy as it is automated. Also, the only solution with our version of docusaurus is a little dirty hack)

### CODE REVIEW

File: documentation/versioned_docs/version-2.x/API Reference.md [hunk 1]
```diff
@@ -9842,8 +9842,10 @@ Represents a measurement of how much geometry can be loaded.
 
 | Name | Type | Description |
 | :------ | :------ | :------ |
+| `geometryDownloadSizeBytes` | `number` | Number of bytes of the geometry that must be downloaded. |
+| `highDetailProximityThreshold` | `number` | Sectors within this distance from the camera will always be loaded in high details. |
+| `maximumNumberOfDrawCalls` | `number` | Estimated maximum number of WebGL draw calls to download geometry for. Draw calls are very important for the framerate. |
 | `maximumRenderCost` | `number` | Maximum render cost. This number can be thought of as triangle count, although the number doesn't match this directly. |
-| `highDetailProximityThreshold` | `number` | For 3D models processed before Q1 2022, sectors within this distance from the camera will always be loaded in high details (deprecated). |
```
Comment (haakonflatval-cognite) [inline documentation/versioned_docs/version-2.x/API Reference.md:9846]: Is this change supposed to be here? I thought this change was made recently

File: examples/src/pages/Migration.tsx [hunk 2]
```diff
@@ -118,7 +118,7 @@ export function Migration() {
       async function addModel(options: AddModelOptions) {
         try {
           const model = options.localPath !== undefined ? await viewer.addCadModel(options) : await viewer.addModel(options);
-
+
```
Comment (haakonflatval-cognite) [inline examples/src/pages/Migration.tsx:121]:
```suggestion
```

File: examples/yarn.lock [hunk 3]
```diff
@@ -2230,6 +2230,11 @@
   dependencies:
     "@types/jest" "*"
 
+"@types/three@0.133.0":
+  version "0.133.0"
+  resolved "https://registry.yarnpkg.com/@types/three/-/three-0.133.0.tgz#2e344ce10bc81dc89295b79a98c09932b3f0c294"
+  integrity sha512-neDN9L64GK1fhT2raWXYIFvIr6ktM6XYgssfTh4zZp+XRgowD1qJ4/w79hLrLQ0w87u1E91ZfvHB0198wBsuHg==
+
```
Comment (haakonflatval-cognite) [inline examples/yarn.lock:2237]: I suggest we don't do any changes in the examples yarn.lock in this PR
```suggestion
```

File: examples/yarn.lock [hunk 4]
```diff
@@ -12169,6 +12174,11 @@ text-table@0.2.0, text-table@^0.2.0:
   resolved "https://registry.yarnpkg.com/text-table/-/text-table-0.2.0.tgz#7f5ee823ae805207c00af2df4a84ec3fcfa570b4"
   integrity sha1-f17oI66AUgfACvLfSoTsP8+lcLQ=
 
+three@0.133.0:
+  version "0.133.0"
+  resolved "https://registry.yarnpkg.com/three/-/three-0.133.0.tgz#41427306c731a89407221b7be48c7a63580400e7"
+  integrity sha512-1p8xTXnJD4hMM2Ktm7+FqOOBoImBoftKnKtAZT/b9AQeL3LhRkVu/HnIJhWA69qIMvUYpjfnunNsO4WdObw1ZQ==
+
```
Comment (haakonflatval-cognite) [inline examples/yarn.lock:12181]: Same reason as above
```suggestion
```

**Review by haakonflatval-cognite — CHANGES_REQUESTED** [review]

**Comment by haakonflatval-cognite** [comment]
General question - why does the PR contain many older commits? Is it based on some branch that has diverged from `release/3.0-alpha`? Or rather - been diverged from?

**Comment by Savokr** [comment]
> General question - why does the PR contain many older commits? Is it based on some branch that has diverged from `release/3.0-alpha`? Or rather - been diverged from?

Not sure why, it was based on release/3.0-alpha. Oh, maybe it's because it was before the force push to 3.0?

File: documentation/versioned_docs/version-2.x/API Reference.md (hunk 1 above)
Comment (Savokr) [reply documentation/versioned_docs/version-2.x/API Reference.md:9846]: This is from master, so shouldn't be a problem, I think. Change is more than 2 months old. I've just used master version for 2.x version.

Comment (larsmoa) [reply documentation/versioned_docs/version-2.x/API Reference.md:9846]: Hmmm, have we removed this in master?

**Comment by larsmoa** [comment]
> > General question - why does the PR contain many older commits? Is it based on some branch that has diverged from `release/3.0-alpha`? Or rather - been diverged from?
> 
> Not sure why, it was based on release/3.0-alpha. Oh, maybe it's because it was before the force push to 3.0?

Maybe we should create a new branch from release/3.0-alpha and cherrypick relevant commits over?

File: documentation/versioned_docs/version-2.x/API Reference.md (hunk 1 above)
Comment (haakonflatval-cognite) [reply documentation/versioned_docs/version-2.x/API Reference.md:9846]: It's not in master, I believe it's only merged into release/3, and will only be in the docs when 3.0 lands

Comment (Savokr) [reply documentation/versioned_docs/version-2.x/API Reference.md:9846]: Well, It's from master in both version 2.x and Next, that's for sure. You can check here - https://cognitedata.github.io/reveal-docs/docs/API%20Reference  by searching `maximumRenderCost`.

Comment (Savokr) [reply documentation/versioned_docs/version-2.x/API Reference.md:9846]: It is possible that it's not in release because of wrong merge

**Comment by Savokr** [comment]
Created a new branch with clean commit history and all changes from this branch

File: documentation/versioned_docs/version-2.x/API Reference.md (hunk 1 above)
Comment (Savokr) [reply documentation/versioned_docs/version-2.x/API Reference.md:9846]: I'll remove this change...

**Pull request closed without merging**